superset-frontend/src/components/Select/CustomTag.tsx

```diff
@@ -69,9 +71,12 @@ export const oneLineTagRender = (props: CustomTagProps) => {
     }
   };
 
-  return (
-    <Tag onMouseDown={onPreventMouseDown} {...props}>
-      {label}
-    </Tag>
-  );
+  if (value !== SELECT_ALL_VALUE) {
+    return (
+      <Tag onMouseDown={onPreventMouseDown} {...props}>
```

Comment:
To get rid of the typescript error:
```suggestion
      <Tag onMouseDown={onPreventMouseDown} {...(props as object)}>
```